elastic/beats
# Fix metricbeat magefile to generate metricbeat/include/fields.go and run make update

## What does this PR do?

This PR updates metricbeat magefile in order to generate metricbeat/include/fields/fields.go. 


## Why is it important?

The fields.go file was not being updated after `make update`.  In case a field was updated in fields which where included in metricbeat  this was  leading to tests failing.

For example updating an existing field in [libbeat/processors/add_kubernetes_metadata/_meta/fields.yml](https://github.com/elastic/beats/blob/master/libbeat/processors/add_kubernetes_metadata/_meta/fields.yml), then [metricbeat/include/fields/fields.go](https://github.com/elastic/beats/blob/master/metricbeat/include/fields/fields.go) was not getting updated.

When tests run they get the available fields from [FieldsRegistry](https://github.com/elastic/beats/blob/86004dd4ba40b40e3e88344e7e4c0e47670886d7/libbeat/asset/registry.go#L32)
which gets filled by the fields.go files. 

So as far as the unit [tests](https://github.com/elastic/beats/blob/86004dd4ba40b40e3e88344e7e4c0e47670886d7/metricbeat/mb/testing/testdata.go#L320) are concerned the fields where not updated, thus leading to failures.

## Checklist




## How to test this PR locally

After making an update in [libbeat/processors/add_kubernetes_metadata/_meta/fields.yml](https://github.com/elastic/beats/blob/master/libbeat/processors/add_kubernetes_metadata/_meta/fields.yml),
run `make update` in metricbeat and notice the metricbeat/include/fields/fields.go getting updated.

## Related issues

- Closes https://github.com/elastic/beats/issues/28021
